docs/source/reading_plans.rst

```diff
@@ -1,123 +1,237 @@
+the :code:`.conf` file when using a date-based plan:
 
-Versification=Vulg
-1=Sir.1-Sir.2
-2=1Macc.1-1Macc.2
-3=Bar.1-Bar.2
+:code:`AndBibleReadingPlanDateBased=true`
 
-Set Start to Another Day
--------------------
-If you started the readings on a previous day and have just switched to AndBible then select Menu from the Daily Reading screen and then select 'Set Start Date...'. You may then jump to 'Move the current day to a specific day' below to skip over readings you have already done.
+Example reading plans can be found
+`here <https://github.com/AndBible/and-bible/tree/master/app/src/main/assets/readingplan>`_.
 
-Abandon Reading Plan
-----------------------------
-Press Menu/Reset from the Daily Reading screen.
+.. note::
 
-Switch between Reading Plans
-----------------------------
-Select the first button in the toolbar when in the Daily Reading screen. By this means it is possible to use several plans simultaneously.
+    `OSIS <https://crosswire.org/osis/>`_ format references and Bible
+    book names must be used in reading plans.
 
-Preview a plans readings
-----------------------------
-When in the Daily Reading screen select the second button in the toolbar.
+    A full list of OSIS book names can be found
+    `here <https://wiki.crosswire.org/OSIS_Book_Abbreviations>`_.
 
-Go to a specific day
-----------------------------
-When in the Daily Reading screen select the second button in the toolbar. Then select the required day from the list.
+.. caution::
+
+    Using an Android version older than 10?
+
+    If you are creating a reading plan to be used on an Android device older than
+    version 10, **and** you plan to include deuterocanonical books in your reading
+    plan, you must specify the versification in the :code:`.properties` file:
+
+    .. code-block::
 
-Move the current day to a specific day
-This is useful if you have been using a reading plan externally from AndBible. A sequence of actions might be i) select plan ii) Set start date to another day (as above) iii) Move the current day to a specific day (as below)
+        Versification=Vulg
+        1=Sir.1-Sir.2
+        2=1Macc.1-1Macc.2
+        3=Bar.1-Bar.2
 
-Version 3.3 and earlier: Go to the day prior to the desired day as described in the above paragraph. Select Menu/Done (not the normal Done button at the bottom) this bypasses checks and marks the current day as Done and the next day will become the current day.
+    The default versification for reading plans is `KJV`.
 
-Note: Starting in 4.0-beta (and 4.0 stable), you will be able to simply go to 3-dot menu on top right, and click "Set as current day".
+.. note::
+    Beginning in version 4.0, the :code:`jsword/readingplan` module format for
+    reading plans is deprecated for use with Android 10+ since the :code:`jsword/readingplan`
+    path no longer works on Android 10+. The new :code:`.zip` format (described above)
+    can be imported like any other |app| module.
```

Comment:
As per other obsolete notes, I don't think this is really needed or useful anymore either.